.github/workflows/test.yml

```diff
@@ -0,0 +1,52 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Run crawlers
+
+on:
+  push:
+    branches: [ master, dev ]
+  pull_request:
+    branches: [ master, dev ]
+
+jobs:
+  python-tests:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run:
+        python -m pip install --upgrade pip
+        pip install Scrapy
+    - name: Delete old JSON files
+      run: |
+        rm -r docs-crawler/docs/output/tf/2.1.json
+        rm -r docs-crawler/docs/output/tfjs/1.5.1.json
+        rm -r docs-crawler/docs/output/torch/1.4.0.json
+    - name: Run all crawlers
+      run: |
+        cd docs-crawler/docs/
+        bash run_crawlers.sh
+    - name: run aggregate_crawler
+      run: python docs-crawler/docs/aggregate_crawler_output.py
+    - name: Commit to repository
+      run: |
+        git config user.name "GitHub Action"
+        git config user.email "actions@users.noreply.github.com"
+        cd pythreepio/static
+        git add  mapped_commands_full.json
+        git commit -m "added final ouput"
+        git push origin master
+        
```

Comment:
This section should be replaced with the `[ $(git status --short | wc -l) = 0 ]` we talked about in slack :)

Reply:
ok

Reply:
I was trying something new

Reply:
I will make the changes